app/templates/suppliers/details.html

```diff
@@ -71,23 +71,28 @@
   caption='Registration information',
   field_headings=[
     'Label',
-    'Value'
+    'Value',
+    'Change'
   ],
   field_headings_visible=False
 ) %}
-{% if supplier.registeredName %}
-  {% call summary.row() %}
-    {{ summary.field_name('Registered company name') }}
-      {{ summary.text(supplier.registeredName) }}
-  {% endcall %}
-{% endif %}
+{% call summary.row() %}
+  {{ summary.field_name('Registered company name') }}
+  {% if supplier.registeredName %}
+    {{ summary.text(supplier.registeredName) }}
+    {{ summary.edit_link(label="Change", link=url_for(".edit_supplier_registered_name")) }}
```

Comment:
Can we use the hidden_text field for these `edit_links` to provide context for screen reader users? For example:

```
summary.edit_link(label="Change", link=url_for(".edit_supplier_registered_name"), hidden_text="Registered company name")
```

See here for more context: https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/804

Reply:
Yes, great idea :)